### PR TITLE
1.9 unset fix

### PR DESCRIPTION
**Pull request by hdwebpros**
Fixed unset bug on 1.9 version. Not sure how to do this cleaner. The two commits I did that are noteworthy are:
https://github.com/getgrav/grav-plugin-admin/commit/135387e0823d8b9bd8d1ecc23df8edb9583a76d4
https://github.com/getgrav/grav-plugin-admin/commit/264967a4dd90abdca82362dad1037b9a71601afc

I have 4 clients that have websites on Grav that use the same images on more than one page. That is why they need unset. If they delete the image, it gets removed from other pages too. Unset just removes the image but allows it to be on other pages. 

The bug right now is that if you have a blueprint with more than one area for images, unset effects everything and sets everything to one image. These commits fix that. Tested locally.

### CODE REVIEW

**Comment by rhukster** [comment]
Why do you have 74 files changed Ryan? Did you start from admin develop branch? Should only have the files you actually modified.

**Comment by w00fz** [comment]
It’s a little bit hard to follow what’s going on and what changed

**Comment by hdwebpros** [comment]
https://github.com/getgrav/grav-plugin-admin/commit/135387e0823d8b9bd8d1ecc23df8edb9583a76d4

And

https://github.com/getgrav/grav-plugin-admin/commit/264967a4dd90abdca82362dad1037b9a71601afc

Do you need me to describe what was done in that commit that made it work?

**Comment by hdwebpros** [comment]
Just following up on this. There are no other PRs that I've seen on this repo that someone had to do against a branch that is not main.

But, I've explained every line in my main commit. I just have no way to do the PR against the 1.9 branch. I researched and I'd have to fetch upstream and do some other git related stuff that I flat out aren't that comfortable with or good at.

I appreciate any assistance here. This fix would help so many clients

**Comment by mahagr** [comment]
@hdwebpros Everything is now merged to develop, so please try again. Github has very good documentation, so please read it, too. :)
